component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1121,8 +1118,8 @@ public Response updateUserProfileAttributes(@Context
         LOG.error("Error updating user {} attributes", currentUser, e);
         return Response.serverError().build();
       }
-
     }
+    identityManager.updateProfile(profile, getCurrentUser(), true);
```

Comment:
What if nothing change ? we save the profil event if there is no modification ?

Reply:
I juste put the updateProfile out of the loop, initially the rest service don't check if the property is changed or not, this check is done in the app level (client side). I'm not sure if this should be handled in this PR.